akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorAdapter.scala

```diff
@@ -121,13 +142,33 @@ import akka.util.OptionVal
     case other               ⇒ super.unhandled(other)
   }
 
-  override val supervisorStrategy = a.OneForOneStrategy() {
+  override val supervisorStrategy = a.OneForOneStrategy(loggingEnabled = false) {
+    case TypedActorFailedException(cause) ⇒
+      // These have already been optionally logged by typed supervision
+      recordChildFailure(cause)
+      a.SupervisorStrategy.Stop
     case ex ⇒
-      val ref = sender()
-      if (context.asInstanceOf[a.ActorCell].isWatching(ref)) failures = failures.updated(ref, ex)
+      recordChildFailure(ex)
+      val logMessage = ex match {
```

Comment:
I copied this from what the untyped supervisor does. We can perhaps reevaluate if we want this logged at error or configurable in a different PR